agent/api/json.go

```diff
@@ -1,224 +0,0 @@
-// Copyright 2014-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
See this [commit](https://github.com/aws/amazon-ecs-agent/commit/1af592561916daae8b0b51a7bda99bea73533e1a)